docs/DeveloperGuide.adoc

```diff
@@ -806,9 +806,59 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
```

Comment:
The target user profile is not updated. Please update it so I can assess its viability or make suggestions for improvements.

Reply:
Will update in my commit